target/armv8m.mk

```diff
@@ -15,17 +15,34 @@ CXX := $(CROSS)g++
 OLVL ?= -O2
 CFLAGS += -mthumb -fomit-frame-pointer -mno-unaligned-access
 
-# TODO hard float perhaps? To be decided
-CFLAGS += -mcpu=cortex-m33 -mfloat-abi=soft -fstack-usage
+# soft FPU for now, no support in kernel for hard FPU
+CFLAGS += -mfloat-abi=soft -fstack-usage
+
+MCX_USE_CPU1 ?= n
+MCX_CPU0_RST_ADDR ?= n
+
+ifeq ($(MCX_USE_CPU1), y)
+  CFLAGS += -mcpu=cortex-m33+nodsp
+else
+  CFLAGS += -mcpu=cortex-m33
+endif
 
 VADDR_KERNEL_INIT := $(KERNEL_PHADDR)
 
 LDFLAGS := -Wl,-z,max-page-size=0x10
 
 ifeq ($(KERNEL), 1)
-  CFLAGS += -ffixed-r9
+  CFLAGS += -ffixed-r9 -DMCX_USE_CPU1
   LDFLAGS += -Tbss=20000000 -Tdata=20000000
   STRIP := $(CROSS)strip
+
+  ifeq ($(MCX_USE_CPU1), y)
+    CFLAGS += -DMCX_USE_CPU1
```

Comment:
Small note:
`-DMCX_USE_CPU1` duplicated in line 35

Reply:
Well, it was pretty important. Thanks for spotting that!